zokrates_cli/src/bin.rs

```diff
@@ -260,9 +260,11 @@ fn cli() -> Result<(), String> {
         ("compile", Some(sub_matches)) => {
             println!("Compiling {}\n", sub_matches.value_of("input").unwrap());
 
-            let path = PathBuf::from(sub_matches.value_of("input").unwrap());
+            let mut path = PathBuf::from(sub_matches.value_of("input").unwrap());
+            path.set_extension("code");
```

Comment:
I think we should replicate python behavior here: when passing a file to the python CLI, you do give the extension afaik
```
python script      
python: can't open file 'script': [Errno 2] No such file or directory
```

Reply:
so I don't think there should be magic happening in the cli, only the import statements. Thoughts welcome though!

Reply:
Ok. I mean currently you can pass the extension, but dont have to. Let me know how I should change it

Reply:
If I read it correctly, if you pass an extension, it overrides it with `.code`. I think at the CLI level, the extension should always be passed, as it is now

Reply:
Ok, will revert those changes